src/components/application_manager/include/application_manager/app_extension.h

```diff
@@ -33,15 +33,18 @@
 #ifndef SRC_COMPONENTS_APPLICATION_MANAGER_INCLUDE_APPLICATION_MANAGER_APP_EXTENSION_H_
 #define SRC_COMPONENTS_APPLICATION_MANAGER_INCLUDE_APPLICATION_MANAGER_APP_EXTENSION_H_
 
+#include <functional>
 #include <memory>
 
-namespace ns_smart_device_link {
-namespace ns_smart_objects {
-class SmartObject;
-}
-}  // namespace ns_smart_device_link
+#include "smart_objects/smart_object.h"
+
+namespace resumption {
+struct ResumptionRequest;
+using Subscriber = std::function<void(const int32_t, const ResumptionRequest)>;
```

Comment:
Leaving my comment from the RC resumption PR here since it is tied to this PR: https://github.com/smartdevicelink/sdl_core/pull/3480#discussion_r476656175

> This `subscriber` callback is quite confusing. I know this comment applies to SDL-0190, but I think this would work much better if it just used an interface with the `SubscribeToResponse` function included, which `ResumptionDataProcessor` could inherit.

Reply:
@jacobkeeler done in https://github.com/smartdevicelink/sdl_core/pull/3475/commits/e601b42dccc527918547dd89edbefddad202ef9b